project/RootBuild.scala

```diff
@@ -1,31 +1,42 @@
-import _root_.sbt.Keys._
-import sbt._
 import sbt.Classpaths.publishTask
-import Keys._
+import sbt.Keys._
+import sbt._
+import sbtassembly.Plugin.AssemblyKeys._
 import sbtassembly.Plugin._
-import AssemblyKeys._
-import scala.sys.process._
-import scala.util.Properties.{ envOrNone => env }
-import scala.collection.JavaConversions._
+
+import scala.util.Properties.{envOrNone => env}
 
 
 object RootBuild extends Build {
 
+  // lazy val root = Project("root", file("."), settings = rootSettings) aggregate(core, spark, examples)
```

Comment:
Remove this line please, may be use `// Project sub modules` for this code block